TLM/TLM/Resources/lang_fr.txt

```diff
@@ -95,26 +95,26 @@ Vehicles_may_do_u-turns_at_junctions Les véhicules peuvent faire demi-tour aux
 Vehicles_going_straight_may_change_lanes_at_junctions Les véhicules allant tout droit peuvent changer de voie aux intersections
 Allow_u-turns Autoriser les demi-tours
 Allow_lane_changing_for_vehicles_going_straight Autoriser le changement de voie pour les véhicules allant tout droit
-Allow_vehicles_to_enter_a_blocked_junction Autoriser les véhicules à entrer dans une jonction bloquée
-Road_condition_has_a_bigger_impact_on_vehicle_speed La condition de la voirie a une plus grand impact sur la vitesse des véhicules
+Allow_vehicles_to_enter_a_blocked_junction Autoriser les véhicules à entrer dans une intersection bloquée
```

Comment:
I prefer the word intersection, although others might disagree.

In English, `Junction` is the nexus of 3+ routes (roads, tracks, whatever) and `Intersection` is a `Stacked junction` (a "junction that uses bridges and/or tunnels"). An intersection is likely to have fewer conflict points than a junction = better rate of traffic flow which is required to handle heavy traffic.

So, erm, just flagging that up in case it needs extra discussion. Like I say, I prefer the word 'intersection' but that's a personal thing.

Reply:
In French, `jonction` is heavily connotated, meaning "a road which connects two highways"
(we mostly talk about `Jonction d'autoroute` where `autoroute `= `highway`).

`Intersection` in French is the generic word, meaning any type of road connection, so what I interpret as being the English `Junction`.

Also, just out of curiosity, is the english `Intersection` really including stacked junctions ?
The wikipedia article describes it at being _at grade_
https://en.wikipedia.org/wiki/Intersection_(road)

Reply:
Ah, in that case intersection is the correct word to use.

In England (I said English but meant England) we generally refer to everything as a junction (including Spaghetti Junction, Magic Roundabout, etc) but there's this sense that anything big and multi-level is an intersection. While technically incorrect, it's de facto language that people use.